aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -131,11 +288,60 @@ var host = new WebHostBuilder()
 > [!NOTE]
 > HTTPS and HTTP cannot be hosted on the same port.
 
+# [ASP.NET Core 2.x](#tab/aspnetcore2x)
+
+Only HTTP URL prefixes are valid; Kestrel does not support SSL when you configure URL bindings by using `UseUrls`.
+
+* IPv4 address with port number
+
+  ```
+  http://65.55.39.10:80/
+  ```
+
+  0.0.0.0 is a special case that binds to all IPv4 addresses.
+
+
+* IPv6 address with port number
+
+  ```
+  http://[0:0:0:0:0:ffff:4137:270a]:80/ 
+  ```
+
+  [::] is the IPv6 equivalent of IPv4 0.0.0.0.
+
+
+* Host name with port number
+
+  ```
+  http://contoso.com:80/
+  http://*:80/
+  ```
+
+  Host names, *, and +, are not special. Anything that is not a recognized IP address or "localhost" will bind to all IPv4 and IPv6 IPs. If you need to bind different host names to different ASP.NET Core applications on the same port, use [WebListener](weblistener.md) or a reverse proxy server such as IIS, Nginx, or Apache.
```

Comment:
* Escape the asterisk \\\*
* Comma splice after the plus sign
* "are not" :point_right: "aren't"
* "is not" :point_right: "isn't"
* "will bind" :point_right: "binds"
😅